guides/doc-Managing_Hosts/topics/proc_registering-a-host.adoc

```diff
@@ -33,83 +33,97 @@ Optional: If the *Registration* feature is not enabled on your {SmartProxy}, ent
+* RHEL 8: `rhel-8-for-x86_64-appstream-rpms`
 +
+The `insights-client` package is installed by default on RHEL 8 except in environments whereby RHEL 8 was deployed with "Minimal Install" option.
 
-. Optional: {Project} uses the JSON Web Tokens (JWT) for authentication.
+. Optional: In the *Token lifetime (hours)* field, change the validity duration of the JSON Web Token (JWT) that {Project} uses for authentication.
 The duration of this token defines how long the generated `curl` command works.
-If you want to change the duration of the token, enter the required duration to the *Token lifetime (hours)* field. You can set the duration to 0 - 999 999 hours or unlimited.
+You can set the duration to 0 - 999 999 hours or unlimited.
 +
-Note that {Project} uses the permissions of the user who generates the `curl` command for authorization of hosts.
+Note that {Project} applies the permissions of the user who generates the `curl` command to authorization of hosts.
 If the user loses or gains additional permissions, the permissions of the JWT change too.
 Therefore, do not delete, block, or change permissions of the user during the token duration.
-Also, the scope of JWT is limited to the registration endpoints only and cannot be used anywhere else.
-. Optional: In the *Remote Execution Interface* field, enter a network interface that hosts must use for the SSH connection.
++
+The scope of the JWTs is limited to the registration endpoints only and cannot be used anywhere else.
+. Optional: In the *Remote Execution Interface* field, enter the identifier of a network interface that hosts must use for the SSH connection.
 If you keep this field blank, {Project} uses the default network interface.
-. Optional: *Install packages* - Install packages on the host when registered. Can be set by `host_packages` parameter
+. Optional: In the *Install packages* field, list the packages (separated with spaces) that you want to install on the host upon registration. This can be set by the `host_packages` parameter.
+. Optional: Select the *Update packages* option to update all packages on the host upon registration. This can be set by the `host_update_packages` parameter.
 
-. Optional: *Update packages* - Update packages on the host when registered. Can be set by `host_update_packages` parameter
-. Optional: *Repository* - A repository to be added before the registration is performed. For example, it can be useful to make the subscription-manager packages available for the purpose of the registration. For Red Hat family distributions, this should be the URL of the repository.
-For example, http://rpm.example.com/.
+. Optional: In the *Repository* field, enter a repository to be added before the registration is performed. For example, it can be useful to make the subscription-manager packages available for the purpose of the registration.
```

Comment:
```suggestion
. Optional: In the *Repository* field, enter a repository to be added before the registration is performed.
For example, it can be useful to make the `subscription-manager` packages available for the purpose of the registration.
```
Does `subscription-manager` consist of multiple packages?

Reply:
@lzap I'm gonna need help on this one, because I have no freaking idea yet :) Thank you!

Reply:
Oh it is only a single package.